server/cpho/management/commands/seed_programs.py

```diff
@@ -0,0 +1,130 @@
+from pathlib import Path
+
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from cpho.models import (
+    DimensionType,
+    DimensionValue,
+    Indicator,
+    IndicatorDatum,
+    Period,
+    PHACOrg,
+    PhacOrgRole,
+    User,
+)
+
+BASE_DIR = Path(__file__).resolve().parent.parent.parent
+
+
+class Command(BaseCommand):
+    """Seeds the HOPIC database with PHAC orgs from CSV and creates admin group"""
+
+    help = "Seeds the HOPIC database with PHAC orgs from CSV and creates admin group"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--mode",
+            type=str,
+            default="upsert",
+            help="Program seeding options: wipe, reset, upsert, insert_ignore. Default is upsert",
+        )
+        parser.add_argument(
+            "--yes",
+            action="store_true",
+            help="Automatically accepts all confirmation prompts (risky!). Default is False",
+        )
+
+    def handle(self, *args, **kwargs):
+        with transaction.atomic():
+            # We always need admin / account admin groups! This is non-destructive
+            # Group.objects.get_or_create(name=ITAP_ADMIN_GROUP)
+            # Group.objects.get_or_create(name=ITAP_ACCOUNT_ADMIN_GROUP)
+
+            # Confirmation prompt
+            if not kwargs["yes"] and kwargs["mode"] in ["reset", "wipe"]:
+                confirm = input(
+                    "This will delete all existing PHAC orgs. Are you sure? (y/N) "
+                )
+                if confirm.lower() != "y":
+                    print("Aborting...")
+                    return
+
+            # Create / update programs
+            seed_programs(kwargs["mode"])
+
+
+def seed_programs(mode):
+    """Seed programs based on options (upsert, reset, insert_ignore)"""
+    # TODO: Currently only supports "reset" and wipe
+    if mode in ["reset", "wipe"]:
+        IndicatorDatum.objects.all().delete()
+        Indicator.objects.all().delete()
+        # Remove existing programs
+        PhacOrgRole.objects.all().delete()
+        PHACOrg.objects.all().delete()
+        print("PHAC orgs wiped")
+        if mode == "wipe":
+            return
+    create_phacorgs(mode)
+
+
+def create_phacorgs(mode="reset"):
+    IndicatorDatum.objects.all().delete()
+    Indicator.objects.all().delete()
+    if mode == "reset":
+        PhacOrgRole.objects.all().delete()
+        PHACOrg.objects.all().delete()
+
+    updated_count = 0
+    inserted_count = 0
+    skipped_count = 0
+    with open(BASE_DIR / "scripts/phac_orgs.csv", encoding="utf-8") as f:
+        reader = DictReader(f)
+        for data in reader:
+            data["pk"] = data["pk"] if data["pk"] != "" else None
+            data["parent_pk"] = (
+                data["parent_pk"] if data["parent_pk"] != "000000" else None
+            )
+
+            is_branch = int(data["pk"]) % 10000 == 0
```

Comment:
Couldn't this just be a `@property` or annotation? (An annotation has the advantage of being available for filtering)

My concern is we'll probably have to uproot this a bit when we start putting PHACOrg into its own helper package, better to not add any columns that aren't already in ITAP 

It's not a huge deal right now, though, it's easy enough to change later